.github/workflows/oas-validation.yml

```
@@ -0,0 +1,14 @@
+on: [push]
```

Cosmetic proposal for better view of the workflow in the Actions tab
```suggestion
name: Validate OpenAPI Specification
on: [push]
```